spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -400,7 +402,7 @@ def step(  # noqa
                     self.spot.set_base_velocity(base_action[0], 0, 0, MAX_CMD_DURATION)
                     target_yaw = None
         elif not (grasp or place):
-            print("!!!! NO ACTIONS CALLED: moving to next step !!!!")
+            self.say("!!!! NO ACTIONS CALLED: moving to next step !!!!")
```

Comment:
There is no need for this change. Let's keep it `print`. No need to emit this as audio-out.